Modules/_elementtree.c

```diff
@@ -2103,6 +2095,8 @@ elementiter_dealloc(ElementIterObject *it)
     Py_XDECREF(it->root_element);
 
     PyObject_GC_Del(it);
+    PyTypeObject *tp = Py_TYPE(it);
+    Py_DECREF(tp);
```

Comment:
You have to get the type before you GC delete the object

Reply:
Yes, thanks!